cmd/machine-api-operator/start.go

```diff
@@ -67,7 +67,7 @@ func runStartCmd(cmd *cobra.Command, args []string) {
 		Callbacks: leaderelection.LeaderCallbacks{
 			OnStartedLeading: run,
 			OnStoppedLeading: func() {
-				glog.Fatalf("leaderelection lost")
+				glog.Fatalf("leader election lost")
```

Comment:
uppercase `Leader`

Reply:
Awkward one, but I tend to go with: https://github.com/golang/go/wiki/CodeReviewComments#error-strings